spec/controllers/admin/subscription_line_items_controller_spec.rb

```diff
@@ -97,7 +98,8 @@
               before { params.merge!(schedule_id: schedule.id) }
 
               context "where no relevant variant override exists" do
-                it "returns a serialized subscription line item with a price estimate, based on the variant" do
+                it "returns a serialized subscription line item with a price estimate, " \
+                   "based on the variant" do
```

Comment:
This isn't pretty either. We could exclude `it` from this rule. Or we reword the test descriptions. Maybe this rule will make people use shorter descriptions in the future. :wink:

Reply:
This fits:
`it "returns serialized subscrip. line item w/ price estimate, based on variant" do`

Reply:
You don't need to change it now. I approved the pull request. 

The comment was about these changes in general, not just this one line. The `it` calls can be wordy and line breaks in strings are ugly. But I'm not sure what the solution is.

Reply:
I tried where possible to shorten them without losing the meaning. Should I go through them again and try to make them shorter?

Reply:
No, just leave it. We'll wait for a second review and can merge then.